server/src/main/java/io/littlehorse/server/streamsimpl/coreprocessors/KafkaStreamsLHDAOImpl.java

```diff
@@ -699,7 +676,7 @@ private void deleteAllChildren(WfRun wfRun) {
         ) {
             while (iter.hasNext()) {
                 LHIterKeyValue<ExternalEvent> next = iter.next();
-                getableStorageManager.delete(next.getKey(), ExternalEvent.class);
+                storageManager.deleteGetable(next.getKey(), ExternalEvent.class);
```

Comment:
what's the difference between deleteGetable and delete?

Reply:
I couldn't find a better name but basically the `delete` works with the new StoredGetable and the `deleteGetable` works with the metadata that's not using the StoredGetable yet